src/nvim/ui_bridge.c

```diff
@@ -305,6 +306,18 @@ static void ui_bridge_update_bg_event(void **argv)
   ui->update_bg(ui, PTR2INT(argv[1]));
 }
 
+static void ui_bridge_update_sp(UI *b, int sp)
+{
+  UI_CALL(b, update_sp, 2, b, INT2PTR(sp));
+}
+static void ui_bridge_update_sp_event(void **argv)
+{
+  UI *ui = UI(argv[0]);
+  if (ui->update_sp) {
```

Comment:
you should add a stub to the tui implementation as is already done for operations that the tui doesn't support.